proposals/source-install.md

```diff
@@ -0,0 +1,1133 @@
+# Installing the AWS CLI v2 from source
+
+Proposal         | Metadata
+---------------- | -------------
+**Author**       | Kyle Knapp
+**Status**       | Proposed
+**Created**      | 24-August-2021
+
+## Abstract
+
+This document proposes an official mechanism to install the AWS CLI v2 from
+source along with an official source distribution artifact.
+
+
+## Motivation
+
+The AWS CLI v2 is available as pre-built executables for macOS,
+Linux x86_64, Linux aarch64, and Windows (64-bit). The AWS CLI v2 is also
```

Comment:
The linux should probably also specify glibc.

Reply:
I have a bullet point for it later on, but I'll see if I can include a note earlier.